src/app/ReadPrepareParams.h

```diff
@@ -48,6 +48,12 @@ struct ReadPrepareParams
     bool mKeepSubscriptions             = false;
     bool mIsFabricFiltered              = true;
     bool mIsPeerLIT                     = false;
+    // If Application knows the peer is LIT ICD, this can be set true. If application has subscribed
+    // ICDManagementCluster::OperatingMode, its operation mode can be set automatically, it is not required to set mIsPeerLIT.
+
+    bool mRegisteredCheckInToken = false;
+    // If Application has registered the check-in token into the peer device, this mRegisteredCheckInToken needs to be true.
```

Comment:
The comments keep referencing true/false with explanations. How about having an enumeration with clearer names instead? If it is based on u8 it would take the same amount of storage.

I also wonder if there are some interactions with the mIsPeerLit ... what are the actual states that a readclient can be in?

Reply:
no other interaction for mIsPeerLit, If application/user has determined this device is LIT, and only wanna subscribe with some attributes except ICDManagementCluster::OperatingMode, it would set mIsPeerLit as true.

subscription's operation model would be mainly determined by ICDManagementCluster::OperatingMode if subscription has  ICDManagementCluster::OperatingMode via wildcard or concrete path,
subscription's operation model would be forcibly staying in normal mode if application has not yet registerred token to device, this is pretty importance if device has been multi-admined into LIT by another ecosystem